[Sprig App] PuzzleMania

SigmaSkibidiNumbers opened the pull request:
### Author name
Author: Johamez Wilzoney

### About your game

**What is your game about?**
A Puzzle game about flipping switches and pushing rocks

There are 5 bonus levels unlocked by a key on a secret level (the portal to said secret level is on lv 2)

**How do you play your game?**
"W A S D" to move
"I" to Mute music
"J" to interact with things
"K" to reset level
"L" for level select

gusruben reviewed (changes requested):
Thanks for your submission, I like it a lot! Unfortunately I seem to be getting an error when flipping the switch in the second level 4: `TypeError: can't access property "x", getFirst(...) is undefined`. If you fix this I'll be happy to re-review!

Also, a very optional suggestion: some indicators for which levels are finished would help a lot with skill issues like mine :)

SigmaSkibidiNumbers commented:
I fixed the issue and changed the fake wall texture slightly to make it a
bit easier to spot, I didnt intend for the error on Level B4, however the
lever was meant to be a trick that forces you to restart the level as it
deletes the portal you need to complete the level.

On Fri, 20 Dec 2024 at 21:23, Gus Ruben ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> Thanks for your submission, I like it a lot! Unfortunately I seem to be
> getting an error when flipping the switch in the second level 4: TypeError:
> can't access property "x", getFirst(...) is undefined. If you fix this
> I'll be happy to re-review!
>
> Also, a very optional suggestion: some indicators for which levels are
> finished would help a lot with skill issues like mine :)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/hackclub/sprig/pull/2722#pullrequestreview-2518358705>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BNDBNOSZT7HYKGAL7C7653L2GSDFPAVCNFSM6AAAAABTYTM2RGVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDKMJYGM2TQNZQGU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

grymmy commented:
@SigmaSkibidiNumbers you stated you fixed the issue that @gusruben pointed out here - can you please link to the commit where this was done?  am I missing something?

gusruben commented:
It seems like this PR is a duplicate of #2622, which is more up-to-date-- check for feedback there!
